### PR TITLE
Remove SSH access to Knative repos for Prow jobs

### PR DESCRIPTION
**Pull request by adrcunha**
The repos are public, SSH access is not necessary anymore.

### CODE REVIEW

**Comment by adrcunha** [comment]
#141 was merged, rebased.

/hold cancel

File: ci/prow/config.yaml [hunk 1]
```diff
@@ -1651,8 +1354,6 @@ periodics:
   name: ci-knative-serving-go-coverage
   agent: kubernetes
   decorate: true
-  ssh_key_secrets:
-  - ssh-knative
   extra_refs:
   - org: knative
     repo: serving
```
Comment (srinivashegde86) [inline ci/prow/config.yaml]: below this there is the perf cron, which also needs to be updated. I cant comment on that line :)

Comment (adrcunha) [reply ci/prow/config.yaml]: That's weird, my local copy was updated and had no ssh entries anymore... Thanks for pointing it out, let me fix it again.

**Pull request merged**